assignments/arrays.js

```diff
@@ -63,32 +63,58 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
```

Comment:
make sure to remember that when using an array it starts in the zero position so if you loop through you have to use a length -1. Or if you know the exact location ask for that location minus one so in this case your code on line 67 would be 
if(inventory[i].id === 32) {
However since we already know the position you could have also done a straight console log like this

console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}` );

instead of looping